controlalgorithm/composite_algorithm.py

```diff
@@ -36,5 +39,5 @@ def composite_algorithm(act_int_temp):
 
 if __name__ == "__main__":
```

Comment:
Do we need this testing section? If we do, can we put it n a test instead? If not, remove it.